public/assets/js/game.js

```diff
@@ -0,0 +1,59 @@
+// default screen, start button to begin game
+// display player health and enemy health
+// game loads, player given choices on next action to take
+// action performed, result shown, next round starts
+
+// =======================================================================================================
+// testing connection to game.html
+function test() {
+    document.getElementById("combatText").innerHTML = ("hello world");
+}
+
+// =======================================================================================================
+// game logic
+
+$(document).ready(function () {
+    // set the health for the player and the boss
+    let playerHealth = 30;
```

Comment:
if you plan on combining player data with the database, you might want to simply check what's in the database at the time